django/db/backends/postgresql_psycopg2/client.py

```diff
@@ -7,13 +11,40 @@ class DatabaseClient(BaseDatabaseClient):
     executable_name = 'psql'
 
     def runshell(self):
+        def escape_pgpass(txt):
```

Comment:
> [escape_pgpass] shouldn't be defined inside the method

Agreed.
I'm not sure about the best convention to use:
Should it be a staticmethod of the class?
Or a _escape_pgpass on the gobal level?
Or just a regular escape_pgpass on the global level?